### PR TITLE
fix: Security scan reported issue fix

### PR DESCRIPTION
**Pull request by SujeethKumarAri**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RDCC-2462

### Change description ###

Security scan reported issue fix

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by SujeethKumarAri** [comment]
Please DO NOT Merge

**Review by shreedhar-hmcts — APPROVED** [review]

File: charts/rd-location-ref-api/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,11 @@ apiVersion: v2
...
-version: 0.0.11
\ No newline at end of file
+home: https://github.com/hmcts/rd-location-ref-api
+version: 0.0.11
+maintainers:
+  - name: Reference Data Team
+dependencies:
+  - name: java
+    version: ~3.4.4
+    repository: '@hmctspublic'
```
Comment (shashank-rastogi) [inline charts/rd-location-ref-api/Chart.yaml:12]: dont u need to add apiVersion: v2??

Comment (SujeethKumarAri) [reply charts/rd-location-ref-api/Chart.yaml:12]: It is already there in line no.1 - https://github.com/hmcts/rd-location-ref-api/blob/0769d8e839630c6a04dd8f2b2b038734d6dafed5/charts/rd-location-ref-api/Chart.yaml#L1

Comment (shashank-rastogi) [reply charts/rd-location-ref-api/Chart.yaml:12]: 👍

**Review by shashank-rastogi — APPROVED** [review]

**Pull request merged**